### PR TITLE
Added Kondunite builder

### PR DESCRIPTION
**Pull request by liiri**
https://pypi.org/project/kondunite/

### CODE REVIEW

**Comment by liiri** [comment]
@googlebot I consent.

File: kondunite/examples/cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+steps:
+
+  - name: 'gcr.io/$PROJECT_ID/kondunite'
+    id: 'Build unified gke k8s yaml'
+    args: ['--target', 'gke', '-o', 'gke.yaml', './k8s_manifests']
+
+  - name: 'gcr.io/$PROJECT_ID/kondunite'
+    id: 'Build unified replicated yaml'
+    args: ['--target', 'repl', '-o', 'replicated.yaml', '--repl', '--repl-registry', 'gcr.io/$PROJECT_ID', '--repl-base', './k8s_manifests/replicated_base.yaml', './k8s_manifests']
```
Comment (sanastos) [inline kondunite/examples/cloudbuild.yaml]: Are you missing a file from this commit? This build step fails.

**Comment by liiri** [comment]
> Ah, looks like you accidentally checked in your .idea directory. Could you remove it?

yeah sorry about that. removed

File: kondunite/cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (leeonlee) [inline kondunite/cloudbuild.yaml]: Are the hyphens here necessary? If not, please remove for consistency

Comment (liiri) [reply kondunite/cloudbuild.yaml]: removed

**Pull request merged**